tools/debugger/debugger.py

```diff
@@ -172,6 +172,7 @@ def do_eval(self, args):
         self.debugger.eval(args)
         self.stop = True
     do_e = do_eval
+    do_p = do_eval
```

Comment:
I would also add a `do_print = do_eval` for consistency. A test should be added as well (or an existing is extended).